sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -82,7 +82,7 @@ object SQLExecution {
           val redactedStr = Utils
             .redact(sparkSession.sessionState.conf.stringRedactionPattern, sqlStr)
           redactedStr.substring(0, Math.min(truncateLength, redactedStr.length))
-        }.getOrElse(callSite.shortForm)
+        }.orElse(queryExecution.sqlText).getOrElse(callSite.shortForm)
```

Comment:
Maybe we can add some rendering mechanism to define the style of `sqlText`

Reply:
What will the sql query look like if it's very long?